python/tvm/relay/frontend/pytorch.py

```diff
@@ -2977,7 +2977,12 @@ def nll_loss(self, inputs, input_types):
     def flip(self, inputs, input_types):
         data = inputs[0]
         axis = inputs[1]
-        return _op.transform.reverse(data, axis=axis[0])
+        for i, ax in enumerate(axis):
+            if i == 0:
+                out = _op.reverse(data, ax)
+            else:
+                out = _op.reverse(out, ax)
```

Comment:
Want to suggest small refactoring:
```suggestion
        out = data
        for ax in axis:
            out = _op.reverse(out, ax)
```